### PR TITLE
developing

### PR DESCRIPTION
**Pull request by Luisafernanda29**
good evening, this is to give you information about my contribution to the repository, I hope you take it into account.

### CODE REVIEW

File: luisa_cortezano/ejercicio.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+```
+Sub recolecta()
```
Comment (EmanuelObredor18) [inline luisa_cortezano/ejercicio.md:2]: Hi @Luisafernanda29. Good Morning. Please check your code, variables aren't initialized like this. You must initialize variables like this: tr = 0, sb = 0, and soon.

File: luisa_cortezano/ejercicio2.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+```
+Sub datos()
+    f = 1
+    sw = True
+    
+    c = Int(InputBox("ingrese numero de cedula"))
+    
+    While sw
```
Comment (LonelyWalker17) [inline luisa_cortezano/ejercicio2.md:8]: The limit variable must be in true so that you can end the loop. If not, the code will run infinitely.

**Review by EmanuelObredor18 — CHANGES_REQUESTED** [review]

**Review by EmanuelObredor18 — CHANGES_REQUESTED** [review]
Please @Luisafernanda29 edit your flowchart. You still need to indicate which of the addresses the program is going to address if the loop is true.

**Review by EmanuelObredor18 — APPROVED** [review]

**Review by Jesusvillanueva07 — APPROVED** [review]
good job @Luisafernanda29

**Review by hguzman — APPROVED** [review]
Ok

**Review by jvidesDev — APPROVED** [review]
Nice job

**Pull request merged**